### PR TITLE
GG-18747 [.NET] Add EE and UE runnable docker image

### PR DESCRIPTION
**Pull request by ashapkin**
Added dynamic assembly loading functionality to runnable application

### CODE REVIEW

File: modules/platforms/dotnet/Apache.Ignite/Config/ArgsAssemblyLoader.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Apache.Ignite.Core;
+
+    /// <summary>
+    /// Loads required assemblies at runtime.
+    /// </summary>
+    public static class ArgsAssemblyLoader
```
Comment (ptupitsyn) [inline modules/platforms/dotnet/Apache.Ignite/Config/ArgsAssemblyLoader.cs]: `internal`

Comment (ashapkin) [reply modules/platforms/dotnet/Apache.Ignite/Config/ArgsAssemblyLoader.cs]: fixed

File: modules/platforms/dotnet/Apache.Ignite/Config/ArgsAssemblyLoader.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        /// <summary>
+        /// Load assemblies if assembly key is present in arguments.
+        /// This is useful for plugin configuration before parsing the app.config file.
+        /// </summary>
+        /// <param name="args">Application arguments in split form.</param>
+        public static void LoadAssembliesIfRequired(IEnumerable<Tuple<string, string>> args)
```
[the remark this answers is not in the document]
Comment (ashapkin) [reply modules/platforms/dotnet/Apache.Ignite/Config/ArgsAssemblyLoader.cs]: fixed

File: docker/apache-ignite-net/Dockerfile [hunk 3]
```diff
@@ -29,4 +29,7 @@ WORKDIR /app
 COPY libs ./libs
 COPY publish ./
 
+# Container port exposure
+EXPOSE 47500
```
Comment (ptupitsyn) [inline docker/apache-ignite-net/Dockerfile]: Should we use same ports as Java-based Dockerfile does?
`EXPOSE 11211 47100 47500 49112`

**Comment by dspavlov** [comment]
GG-18747 | Closed
-- | --

**Pull request closed without merging**